StructureDefinition/England-Immunization.xml

```diff
@@ -0,0 +1,347 @@
+      <binding>
+        <strength value="extensible" />
```

Comment:
Is there a reason for extensible? UK Core standard is for SNOMED based to be preferred

Reply:
Binding Strength changed to preferred.